src/test/java/com/readup/server/auth/factory/OAuth2UserInfoFactoryTest.java

```diff
@@ -0,0 +1,66 @@
+package com.readup.server.auth.factory;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import com.readup.server.auth.dto.GoogleOAuth2UserInfo;
+import com.readup.server.auth.dto.KakaoOAuth2UserInfo;
+import com.readup.server.auth.dto.NaverOAuth2UserInfo;
+import com.readup.server.auth.dto.OAuth2UserInfo;
+import com.readup.server.auth.exception.UnsupportedOAuthProviderException;
+
+class OAuth2UserInfoFactoryTest {
+
+	private OAuth2UserInfoFactory oAuth2UserInfoFactory = new OAuth2UserInfoFactory();
+
+	@Test
+	@DisplayName("구글 oauth2 유저 인포 반환하기")
+	void test_google_oAuth2UserInfo_provider_success() {
+		Map<String, Object> attributes = new HashMap<>();
+		attributes.put("sub", "readup");
+		attributes.put("email", "readup@readup.com");
+
+		OAuth2UserInfo oAuth2UserInfo = oAuth2UserInfoFactory.getOAuth2UserInfo("google", attributes);
+		assertTrue(oAuth2UserInfo instanceof GoogleOAuth2UserInfo, "반환된 userInfo가 GoogleOAuth2UserInfo 여야 함.");
```

Comment:
```suggestion		
		assertInstanceOf(GoogleOAuth2UserInfo.class, oAuth2UserInfo, "반환된 userInfo가 GoogleOAuth2UserInfo 여야 함.");
```
assert 단순화 추천이 있었는데 확실히 단순화한 표현이 직관적이고 더 좋은 것 같습니다.

Reply:
오 넵넵!